### PR TITLE
use buoy to help manage releases for knative

### PR DESCRIPTION
**Pull request by n3wscott**
Check it out! Let me know what you think.

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -12,35 +12,46 @@ set -o pipefail
...
+    --k8s-release) shift; K8S_VERSION="$1";;
     *) abort "unknown option ${parameter}" ;;
   esac
   shift
 done
-readonly GO_GET
+readonly UPGRADE
+readonly VERSION
+
+#     -require="${dep}@${K8S_VERSION}" \
```
Comment (vaikas) [inline hack/update-deps.sh:39]: remove?

**Review by vaikas — DISMISSED** [review]

File: hack/update-deps.sh [hunk 2]
```diff
@@ -12,35 +12,46 @@ set -o pipefail
...
+#    -replace="${dep}=${dep}@${K8S_VERSION}"
+
+if (( UPGRADE )); then
+  K8S_DEPS=( $(run_go_tool tableflip.dev/buoy buoy float ${ROOT_DIR}/go.mod --domain k8s.io --release ${K8S_VERSION} --strict) )
```
Comment (mattmoor) [inline hack/update-deps.sh:43]: We shouldn't do K8s

**Comment by n3wscott** [comment]
I have a better way, inbound :D

**Comment by n3wscott** [comment]
K8s removed.

**Review by mattmoor — APPROVED** [review]

**Pull request merged**